ci/audit.yml

```diff
@@ -0,0 +1,19 @@
+---
+
+platform: linux
+
+image_resource:
+  type: docker-image
+  source:
+    repository: onsdigital/dp-concourse-tools-node-go
+    tag: latest
+
+inputs:
+  - name: dp-developer-site
+    path: go/src/github.com/ONSdigital/dp-developer-site
+
+outputs:
+  - name: build
+
+run:
+  path: go/src/github.com/ONSdigital/dp-developer-site/ci/scripts/audit.sh
```

Comment:
See comment above about the path.

Reply:
Have updated this as per the modules guide and removed the go route prefix from the path